gearcatalog/frontend/tsconfig.json

```diff
@@ -7,12 +7,12 @@
       },
     ],
     "outDir": "./static/frontend",
-    "sourceMap": true,
+    "sourceMap": false,
```

Comment:
Source maps are quite helpful for debugging. You may want to keep it enabled.

Reply:
Yeah, I reinstalled them on webpack and forgot to reenable them here. Changed.